pkgs/servers/nosql/influxdb2/no-deny-warnings.patch

```diff
@@ -1,10 +0,0 @@
-diff --git a/flux/src/lib.rs b/flux/src/lib.rs
-index 3fdf4071..a4c02277 100644
---- a/flux/src/lib.rs
-+++ b/flux/src/lib.rs
-@@ -1,5 +1,3 @@
--#![cfg_attr(feature = "strict", deny(warnings, missing_docs))]
```

Comment:
Side question: should/Could we perhaps use `--cap-lints warn` by default for rust crates in nixpkgs?

Reply:
That seems like a good idea, would it go in `NIX_RUSTFLAGS` somewhere?